scenarioo-client/app/build/IMainDetailsSection.ts

```diff
@@ -0,0 +1,10 @@
+import {LabelConfigurationMap} from '../shared/services/labelConfigurationsResource.service';
+
+export interface IMainDetailsSection {
+    name: string;
+    key: string;
+    dataTree: any;
+    isFirstOpen: boolean;
+    detailSectionType: string;
+    config?: LabelConfigurationMap;
+}
```

Comment:
DIP: the interface should belong to the detailsarea component that provides this interface as a an interface for all clients of the component and thus this file has to go into the same folder where the detailsarea component is.

Reply:
done